views/includes/date_of_birth-display.html

```diff
@@ -1,4 +1,4 @@
-{% if date_of_birth["day"] == "01" or dob_day == "01" %}
+{% if date_of_birth["day"] == "01" or date_of_birth["day"] == "1" or dob_day == "01" %}
```

Comment:
should we play safe and test dob_day == "01" too?

Reply:
You mean dob_day == **"1"** ?